<ui version="4.0" >
 <class>MainWinOptionsClass</class>
 <widget class="OptionsPageI" name="MainWinOptionsClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>272</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>MainWinOptions</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <spacer name="verticalSpacer" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNoToolbar" >
     <property name="text" >
      <string>Hide toolbar</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkNoFrame" >
     <property name="text" >
      <string>Hide frame</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkRoundCorners" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Round corners</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkToolWin" >
     <property name="text" >
      <string>Tool window (don't show in taskbar)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="chkOnTop" >
     <property name="text" >
      <string>Always on top</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Transparency (%):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="sldTrans" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition" >
        <enum>QSlider::TicksBothSides</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
       <property name="text" >
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2" >
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>OptionsPageI</class>
   <extends>QWidget</extends>
   <header>options_i.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>sldTrans</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>363</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel" >
     <x>389</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkNoFrame</sender>
   <signal>toggled(bool)</signal>
   <receiver>chkRoundCorners</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>58</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel" >
     <x>60</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
